packages/salesforcedx-vscode-apex/package.json

```diff
@@ -25,11 +25,11 @@
     "Programming Languages"
   ],
   "dependencies": {
-    "@salesforce/apex-node": "../../../salesforcedx-apex/packages/apex-node",
+    "@salesforce/apex-node": "^1.0.0",
     "@salesforce/apex-tmlanguage": "1.4.0",
-    "@salesforce/core": "^3.13.0",
+    "@salesforce/core": "^3.23.2",
     "@salesforce/salesforcedx-utils-vscode": "54.12.0",
-    "@salesforce/source-deploy-retrieve": "6.0.3",
+    "@salesforce/source-deploy-retrieve": "6.0.4",
```

Comment:
this bump gets us where we need to be for SDR. Sort of lead to needing to bump all the things.